fastai/optimizer.py

```diff
@@ -44,17 +68,23 @@ def set_hyper(self, k, v):
         assert len(v) == len(self.hypers), f"Trying to set {len(v)} values for {k} but there are {len(self.param_lists)} parameter groups."
         self._set_hyper(k, v)
 
+    def unfreeze(self): self.freeze_to(0)
     @property
     def param_groups(self): return [{**{'params': pg}, **hp} for pg,hp in zip(self.param_lists, self.hypers)]
     @param_groups.setter
-    def param_groups(self, v):
+    def param_groups(self,
+        v  # List of dicts to set `params` and other hyper parameters
```

Comment:
Missing type annotations. Also "hyperparameters"

Reply:
Should type be dict only? `v:dict`?

Reply:
Yeah I think it's just a dict.